CONTRIBUTING.md

```diff
@@ -81,3 +81,23 @@ go run cmd/atc/*.go -d
 ```
 
 Concourse should be live at http://localhost:8080
+
+
+## Building the fly command line tool
+
+If you try to use a realse version of `fly` you will get errors like this:
```

Comment:
This should say something like "If you try to use a released version of `fly` against a development version of `atc`, you will get the following error:"